app/pages/IdentityIssuance/PickProvider/PickProvider.tsx

```diff
@@ -180,6 +220,7 @@ export default function IdentityIssuanceChooseProvider({
                     ledgerCallback={withLedger}
                     preCallback={getIdentityNumber}
                     disabled={!provider}
+                    identityVersion={0}
```

Comment:
Why does this have to be hardcoded to `0`? I would expect that we would use `1` for new identities.

Reply:
This is the version to give to the ledger, and for identity issue we have to get the seed always, because we check whether the index has already been used by any of the versions.